packages/@aws-cdk/aws-secretsmanager/lib/secret.ts

```diff
@@ -160,6 +180,62 @@ export interface ReplicaRegion {
   readonly encryptionKey?: kms.IKey;
 }
 
+/**
+ * An experimental class used to specify an initial secret value for a Secret.
+ * The class wraps a simple string (or JSON representation) in order to provide some safety checks and warnings
+ * about the dangers of using plaintext strings as initial secret seed values via CDK/CloudFormation.
+ */
+export class SecretStringValueBeta1 {
+
+  /**
+   * Creates a `SecretStringValueBeta1` from a plaintext value.
+   * This approach is inherently unsafe, as the secret value will be visible in plaintext in the resulting
+   * CloudFormation template, including in the AWS Console or APIs.
```

Comment:
It will also be visible in the source code, which is also considered a bad practice.

Reply:
It _may_ also be visible in the source code.

Reply:
Yes, still worth mentioning